source/documentation/getting-started.html.md

```diff
@@ -1052,6 +1052,113 @@ Result:
 ManageIQ refreshes the details for the chosen cloud providers.
 
 
+## Containers
+
+The currently supported container provider types are:
+
+- [Kubernetes](http://kubernetes.io/)
+- [OpenShift Origin](http://www.openshift.org/)
+- [OpenShift Enterprise](https://enterprise.openshift.com/)
+- [Atomic](http://www.projectatomic.io/)
+- [Atomic Enterprise](https://access.redhat.com/products/red-hat-atomic-enterprise-platform)
+
+### Service Accounts
+
+To add a Kubernetes or OpenShift provider you must create, in those clusters, a specific `management` service account with the proper role,
+permissions, and authentication token.
+
+For more information on these topics, check out the relevant documentation:
+
+- [Kubernetes Service Accounts](https://github.com/kubernetes/kubernetes/blob/release-1.1/docs/design/service_accounts.md)
+- [Kubernetes Authentication](https://github.com/kubernetes/kubernetes/blob/release-1.1/docs/admin/authentication.md)
+- [OpenShift Service Accounts](https://docs.openshift.com/enterprise/3.0/dev_guide/service_accounts.html)
+
+To add a `management` service account in an OpenShift cluster:
+
+    $ oadm new-project management-infra --description="Management Infrastructure"
+
+    $ oc create -n management-infra -f - <<EOF
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: management-admin
+    EOF
+
+    $ oc create -f - <<EOF
+    apiVersion: v1
+    kind: ClusterRole
+    metadata:
+      name: management-infra-admin
+    rules:
+    - resources:
+      - pods/proxy
+      verbs:
+      - '*'
```

Comment:
You should open an issue to re-visit this after https://github.com/openshift/origin/issues/5973 is resolved so that you don't keep this role forever.